### ISSUE TITLE
Updated code to new version 0.7.9

### ISSUE THREAD

**Issue by Densdix** [u1qqlv/SURVIV.IO-PRO-HACK#23]
Hi, I updated your code to the new version (https://github.com/Densdix/SURVIV.IO-PRO-HACK), thanks for the source code, also added an on / off aimbot. But for some reason, changing the color of textures does not work, and the rest works. Thanks again for the source code.

**Comment by u1qqlv** [comment]
that's so cool that you update the hack, does it freeze? if it does I think I know how to make it stop but you will need to make the code obfuscated too

**Comment by u1qqlv** [comment]
you should submit a patch so I could update mine but good work anyway it works!

**Comment by Densdix** [comment]
Thanks, It sometimes has freeze, I still have not figured out for what reason. But this does not interfere with the game. 
I based on the code downloaded from surviv.io/js/app.*.js (https://github.com/Densdix/SURVIV.IO-PRO-HACK/blob/master/app.6e91a13a.js). From your file, I cut the code that you wrote and pasted there. I saved all the code written by you, the part that does not work, I fixed, the other part so far has commented. Added aim off and today added transparent smoke. 

This is your code, I just migrated to the new version and fixed a few errors, so feel free to take it as a basis. As for abfuscation, I have not yet thought about it, but I can do this at any time.
